### DISCUSSION TITLE
Why vuepress next build is so long ?

### DISCUSSION THREAD

**Discussion by MisTraleuh** [vuepress/core#1166] [Q&A]
Hi, I've a question for everybody,

I've a documentation with **~9000 Markdown files**. And my building take 1h30, is so long no ?

I use `"vuepress": "^2.0.0-beta.49"` and this command for build `"docs:build": "vuepress build docs"`

**How Can I Get A build faster ?**

Thanks you

**Comment by Mister-Hope** [comment]
Maybe you can try vitepress. For your case, it means we build 100 pages per minute, I think that's fine.
